scripts/process_extraction_schemas_json

```diff
@@ -70,9 +134,10 @@ cream and pop tarts). The quantifiers you may see are:
 Take for example the following block:
 
 ```
-Concept/LeafNode
-- arg1 (LeafNode1|LeafNode2) ()
-- [arg2] (LeafNode3) (?)
+-Concept
```

Comment:
In this example, add a space after the dashes `-`, as that reflects the actual output more closely.